README.rst

```diff
@@ -58,7 +58,7 @@ Add cache by decorating a function or method.
     from redis import StrictRedis
     from werkzeug.contrib.cache import RedisCache as _RedisCache
 
-    import cachelper
+    from cachelper import HelperMixin
 
     class RedisCache(_RedisCache, HelperMixin):
```

Comment:
Oh yeah, one comment about this line. A mixin normally goes first, so it can override any attrs/methods provided by both parents. In this case you're not expecting any overlap - the example works either way round. I just mention it, and you can decide which way round you'd like it, to best inform the user who may be using this helper with other cache classes.

Reply:
Thanks for pointing this out.
Yes, this is an issue when using multiple base classes. I'm not sure if users want `HelperMixin` to override other methods with the same names on the other base classes.
Do you think we can make it more explicit by exposing a class decorator instead?

Reply:
Yes I can see the dilemma. You almost want to error upon conflict. But given the classes involved here are cache backends rather than completely arbitrary user written classes, perhaps just a note in the docs along the lines of: _This helper class won't work if the cache backend class already defines methods `call`, `map`, `__call__`._

Reply:
Sure, I'll add this line in the document explicitly.

Reply:
:+1:

Reply:
FYI, I just updated the README and add the note you suggested.
I also released a new version adding support for using functions as key templates.

Reply:
Nice one!